src/ResultWriter/FaultWriterC.cpp

```diff
@@ -43,15 +43,26 @@
 extern "C"
 {
 
-void fault_hdf_init(const int* cells, const double* vertices,
-		int nCells, int nVertices,
-		int* outputMask, const double** dataBuffer, const char* outputPrefix,
-		double interval, char const* xdmfWriterBackend)
+void fault_hdf_init(const int* cells,
```

Comment:
is it only formatting changes?
Better uncommit that?

Reply:
(sorry I know it is annoying)

Reply:
why? The old formatting was difficult to read.

Reply:
because it is then difficult to track changes?
We could automate the formatting at some point as we did for PUMGEN?
I had a similar review in another PR.

Reply:
anyway, that was just my 2 cents, you got my approval if you don't want to change it.

Reply:
>because it is then difficult to track changes?

Really?

>  We could automate the formatting at some point as we did for PUMGEN?

I also proposed clang-format for SeisSol a long ago but it was rejected (or it is better to say postponed). But it is inevitable and at tome point it is going to be introduced. I am personally for introducing clang-format to SeisSol as soon as possible. I know that it will introduce lot's of changes in one commit and lot's people will complain whose branches are a bit behind of the master. But on the other hand, there will never be perfect time to seamlessly introduce clang-format to SeisSol.